src/lib/swapper/swappers/CowSwapper/cowExecuteTrade/cowExecuteTrade.ts

```diff
@@ -128,12 +163,18 @@ export async function cowExecuteTrade(
    * orderId: Orders can optionally include a quote ID. This way the order can be linked to a quote and enable providing more metadata when analyzing order slippage.
    * }
    */
-  const maybeOrdersResponse = await cowService.post<string>(`${apiUrl}/v1/orders/`, {
-    ...orderToSign,
-    signingScheme: SIGNING_SCHEME,
-    signature,
-    from: trade.receiveAddress,
-  })
-
-  return maybeOrdersResponse.andThen(({ data: tradeId }) => Ok({ tradeId }))
+  const baseUrl = getConfig().REACT_APP_COWSWAP_BASE_URL
+  const maybeOrdersResponse = await cowService.post<string>(
+    `${baseUrl}/${network}/api/v1/orders/`,
+    {
+      ...orderToSign,
+      signingScheme: SIGNING_SCHEME,
+      signature,
+      from: trade.receiveAddress,
+    },
+  )
+
+  return maybeOrdersResponse.andThen(({ data: tradeId }) =>
+    Ok({ tradeId, sellAssetChainId: sellAsset.chainId }),
```

Comment:
I get why we need a `ChainId`, that is to get a usable `ChainId` in `cowGetTradeTxs`.
We do happen to get said ChainId from the sell asset, but that's an implementation detail, for all intents and purposes, this property should be `chainId` - at this point, we got a valid trade ID back from CoW, meaning the buy asset and sell assets were correctly on the same chain so this passed validation.

Reply:
fair point

Reply:
https://github.com/shapeshift/web/pull/4595/commits/0cc0dddcddf5cfa71691efc9782e74a74d34de92#diff-69fdfd706315fd0c9916f07834d073ae27ea7469a78f17038333c34b442d1e6d